application/models/Survey.php

```diff
@@ -1165,10 +1165,11 @@ public function getRunning()
             $bExpired = ($oStop < $oNow);
             $bWillRun = ($oStart > $oNow);
 
-            $sStop = convertToGlobalSettingFormat($sStop);
+            $sStop = $sStop != null ? convertToGlobalSettingFormat($sStop) : null;
```

Comment:
For the future, we should probably use `!==`  and `===` instead of the automatically type-casting `!=` or `==`